tensorflow/contrib/android/java/org/tensorflow/contrib/android/TensorFlowInferenceInterface.java

```diff
@@ -370,9 +371,12 @@ public void fetch(String outputName, ByteBuffer dst) {
   private void loadGraph(InputStream is, Graph g) throws IOException {
     final long startMs = System.currentTimeMillis();
 
-    Trace.beginSection("initializeTensorFlow");
+    if( VERSION.SDK_INT >= 18 ){
```

Comment:
TF style is spaces on the outside of parentheses, not the inside.

Also you might as well remove the linebreaks between the Trace calls, since the nesting messes with the previous grouping.